### DISCUSSION TITLE
Loading robots issue

### DISCUSSION THREAD

**Discussion by eduardsik** [visose/Robots#99] [Q&A] [answered]
Hi, I have this message while I try to load robots. The robots folder are under ~/User/Robots/ and using Rhino 6 under mac. The bartlett.xml and 3dm are loaded. 
<img width="894" alt="Screenshot 2020-09-20 at 2 45 14" src="https://user-images.githubusercontent.com/4760893/93691868-ba6a2b00-faeb-11ea-800f-883a102fc293.png">

**Comment by visose** [comment, marked as answer]
You didn't download the xml correctly, search the closed issues as it's a common mistake.
